### PR TITLE
Support for the monomial representation of Newhall polynomials

### PR DESCRIPTION
**Pull request by pleroy**

### CODE REVIEW

File: numerics/newhall.mathematica.h [hunk 1]
```diff
@@ -9,7 +9,7 @@ namespace principia {
 namespace numerics {
 
 FixedMatrix<double, 3 + 1, 2 * 8 + 2> constexpr
```
Comment (eggrobin) [inline numerics/newhall.mathematica.h]: Prefix constexpr, see #1086.

File: mathematica/newhall.nb [hunk 2]
```diff
@@ -346,7 +346,44 @@ Cell[BoxData[
    RowBox[{"NewhallC2", "[", 
     RowBox[{"degree", ",", "divisions", ",", "w"}], "]"}]}]}]], "Input",
  CellChangeTimes->{{3.640856537054928*^9, 3.6408566025854797`*^9}, 
-   3.6408567878820076`*^9}]
+   3.6408567878820076`*^9}],
+
+Cell["\<\
+This function expresses C in a way that is suitable for obtaining the \
```
Comment (eggrobin) [inline mathematica/newhall.nb]: C should be in TraditionalForm. Several other instances of missing TraditionalForm.

File: numerics/newhall.mathematica.h (hunk 1 above)
Comment (pleroy) [reply numerics/newhall.mathematica.h]: Done.

File: mathematica/newhall.nb (hunk 2 above)
Comment (pleroy) [reply mathematica/newhall.nb]: Done.

**Pull request merged**